lib/virt_feature_test_base.pm

```diff
@@ -110,7 +114,13 @@ sub junit_log_params_provision {
     my $start_time = $self->{"start_run"};
     my $stop_time = $self->{"stop_run"};
     $self->{"test_time"} = strftime("\%H:\%M:\%S", gmtime($stop_time - $start_time));
-    $self->{"product_tested_on"} = script_output("cat /etc/issue | grep -io -e \"SUSE.*\$(arch))\" -e \"openSUSE.*[0-9]\"");
+    if (!version_utils::is_alp) {
+        $self->{"product_tested_on"} = script_output("cat /etc/issue | grep -io -e \"SUSE.*\$(arch))\" -e \"openSUSE.*[0-9]\"");
+    } else {
+        alp_workloads::kvm_workload_utils::exit_kvm_container;
+        $self->{"product_tested_on"} = script_output(q@cat /etc/os-release |grep PRETTY_NAME | sed 's/PRETTY_NAME=//'@);
+        alp_workloads::kvm_workload_utils::enter_kvm_container_sh;
+    }
```

Comment:
For testing kvm container on ALP (or any other operation system on host), I think the "product_tested_on" should have more info, because there are host os and container which are two elements tests run with.

Reply:
That's true. 
I thought about this. Actually this parameter "product_tested_on" is kind of redundant info. We can get that from host installation, settings, and testsuite name already. I see that for junit log,  it is better also be there for completeness of results. In real work, QA rarely checks that. 
Besides, given that kvm container in theory can run on  many products' host, so what host is used in test is more interested info for QA. About the container's pesudo os which is less interested, we can also know that from registry, build service, and uploaded log. 
So overall, to be aligned with existing junit log, just add the real host info to let QA know which host it is run on.

Reply:
This is not big deal. If you insist, I can open new ticket to fix all this info for alp, including concurrent_guest_installxxx and others. But in another PR with low priority.

Reply:
This is really not a big deal. You can double check before taking any action, for example, whether such information is important. It looks to me that KVM container is a relatively standalone entity, so product_tested_on should be host_info + container_info if it is supposed to have such info.

Reply:
ok